src/Movies.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export class Movies extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      selectedValue: "",
+      show: false,
+    };
+  }
+
+  componentDidMount() {
+    this.getMovies();
+  }
+
+  getMovies = async () => {
+    let { data } = await axios.get("https://ghibliapi.herokuapp.com/films");
+    this.setState({
+      movies: data,
+    });
+  };
+
+  handleChange = (e) => {
+    if (e.target.value === "") {
+      this.setState({
+        show: false,
+      });
+    } else {
+      this.setState({
+        selectedValue: e.target.value,
+        show: true,
+      });
+    }
+  };
+
+  render() {
+    const { selectedValue, movies, show } = this.state;
+    const options = movies.map((elem, i) => {
+      return (
+        <option key={i} value={elem.title}>
+          {elem.title}
+        </option>
+      );
+    });
+    const movie = (data) => {
```

Comment:
You don't need to pass data to this movie function. You can access movies directly from state instead of passing it as an argument on line 69.